### DISCUSSION TITLE
What does the Binary Input indicate on Philips Hue remote?

### DISCUSSION THREAD

**Discussion by codyc1515** [zigpy/zha-device-handlers#2890] [Q&A]
Philips Hue RWL first gen

https://github.com/zigpy/zha-device-handlers/blob/dev/zhaquirks/philips/rwlfirstgen.py#L7

**Comment by TheJulianJES** [comment]
It just a cluster they expose. It does nothing though. With quirks v2 and ZHA, we should be able to stop the entity from being created automatically.
